arches/app/media/js/viewmodels/card.js

```diff
@@ -332,6 +332,9 @@ define([
             }
         });
 
+        // this.childless = ko.pureComputed(function() {
```

Comment:
@whatisgalen can you please remove this commented out code, as it seems to be cruft